src/libostree/ostree-repo-checkout.c

```diff
@@ -643,6 +647,32 @@ checkout_one_file_at (OstreeRepo                        *repo,
 
       need_copy = FALSE;
     }
+  else if (is_overlayfs_whiteout)
+    {
+      const char *name = destination_name + (sizeof (OVERLAYFS_WHITEOUT_PREFIX) - 1);
```

Comment:
ok, next update should refactor this out to a function where we handle selinux policy handling into the xattrs, and overwrite mode handling.